src/main/java/cz/fi/muni/pa165/Model/Entity/ReturnRecord.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cz.fi.muni.pa165.Model.Entity;
+
+import javax.persistence.Column;
+import javax.persistence.OneToOne;
+import javax.validation.constraints.NotNull;
+import org.springframework.util.Assert;
+
+/**
+ *
+ * @author charlliz
+ */
+public class ReturnRecord extends CarAuditLogItem{
+    
+    @NotNull
+    @Column(nullable = false)
```

Comment:
Some JPA providers can recognize JSR 303 annotations, but it's not guaranteed. Therefore it's recommended to use both.